### PR TITLE
fix:  Solve crashes for API level less than 21

### LINKED ISSUE #1495: Fix App crashes for API level below 21
**Actual Behaviour**

There are many instances where API level below 21 causes crashes in the app.

**Expected Behaviour**

Remove the crashes strictly for API levels below 21.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1495 

Changes: Fixes for API level less than 21.

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -76,6 +77,7 @@ class ChatActivity : AppCompatActivity(), IChatView {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_chat)
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
```
[the remark this answers is not in the document]
Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: vectorDrawables.useSupportLibrary = true
is already set in build.gradle but still it cannot load vector resources

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Use app:srcCompat

[the remark this answers is not in the document]
Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: @iamareebjamal the problem is that while loading a vector drawable as a placeholder using Picasso it causes crash if we do not put the Above statement in the chatactivity and the support was added for the viewholders

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Then add it in one place - Application

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 2]
```diff
@@ -32,6 +33,7 @@ public static MainApplication getInstance() {
     @Override
     public void onCreate() {
         super.onCreate();
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: Add this in a static block above

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ImageViewHolder.java [hunk 3]
```diff
@@ -70,7 +71,7 @@ public void setView(ChatMessage model) {
             try {
                 Picasso.with(itemView.getContext())
                         .load(imageURL)
-                        .placeholder(R.drawable.ic_susi)
+                        .placeholder(ContextCompat.getDrawable(itemView.getContext(),R.drawable.ic_susi))
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ImageViewHolder.java]: Space after comma

You yourself can see these kind of changes

**Pull request merged**